website/pages/ar/managing/deprecate-a-subgraph.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Deprecate a Subgraph
+---
+
+## Deprecating a Subgraph
+
+Although you cannot delete a subgraph, you can deprecate it on Graph Explorer.
+
+### Step-by-Step
```

Comment:
```suggestion
Although you cannot delete a subgraph, you can deprecate it on Graph Explorer.

## Step-by-Step
```

Reply:
Oops, I just realized I left this comment for the file in the `ar` locale... well, I guess it applies to all of them. 😬